app/app.js

```diff
@@ -225,19 +225,21 @@ angular.module('app', ['angular-md5'])
 
       var prevComicIndexInCollection = currentComicIndexInCollection;
       var nextComicIndexInCollection = currentComicIndexInCollection;
+      var LastComicInCollection = vm.prevCollection.comicIds.length;
       prevComicIndexInCollection--;
       nextComicIndexInCollection++;
+      LastComicInCollection--;
 
       // Find the previous comic
-      if (currentComicIndexInCollection > 0) {
-        vm.prevComic = vm.collections[currentCollectionIndexInCollections].comics[prevComicIndexInCollection];
-      } else if (vm.prevCollection) {
+      if(_.first(vm.collection && vm.prevCollection)){ 
```

Comment:
How is this working? Does `first` accept this syntax?

Reply:
No that was just wrong.  It's better now.